analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -323,7 +324,26 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
             val pomArtifact = mavenRepositorySystem
                     .createArtifact(it.groupId, it.artifactId, it.version, "", "pom")
 
-            projectBuilder.build(pomArtifact, projectBuildingRequest).project
+            try {
+                projectBuilder.build(pomArtifact, projectBuildingRequest).project
+            } catch (e: ProjectBuildingException) {
+                e.showStackTrace()
+
+                val id = "${it.groupId}:${it.artifactId}:${it.version}"
```

Comment:
Can we use our `Artifact.identifier()` extension function here?

Reply:
Done.